src/main/resources/templates/c_plus_plus/test/tests/TestClangFormat.py

```
@@ -0,0 +1,57 @@
+    def _run(self) -> None:
+        self.pWrap = self._createPWrap(
+            [self.executable, self.filePath], self.executionDirectory
+        )
+        self._startPWrap(self.pWrap)
+
+        self.pWrap.waitUntilTerminationReading()
+
+        retCode: int = self.pWrap.getReturnCode()
+        if retCode != 0:
+            self._failWith(
+                f"clang-format for file {self.filePath} failed. Returncode is {retCode}."
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)